app/views/host_mailer/pre_tea_time_nudge.markerb

```diff
@@ -1 +1,31 @@
-Template nudge email
+<% if @tea_time.attendees.present? %>?
+  Hey! Your tea time is on Thursday at 7:00pm. Now would be a good time to email your attendees to confirm they're in, offer a chance for them to cancel if needed, and share whatever other info you might need to. Try to do this today so they have sufficient notice.
+
+  <%= @tea_time.attendees.map(&:name).join(", ") %>  
+  <%= @tea_time.attendees.map(&:email).join(", ") %>
+
+  Some of these folks are super trusting and gave you their numbers. Be cool and use them appropriately. Texting them the day of tea time makes it easier for them to remember you're not all that strange (and that helps ensure they show up with good spirits!)
+
+<% if @shareable_num_string %>
```

Comment:
This if statement should be surrounding the previous paragraph as well, not just the string of numbers. If there are no phone numbers, that paragraph makes no sense